spec/views/holds/index.html.erb_spec.rb

```diff
@@ -7,56 +7,12 @@
 
   before do
     allow(hold).to receive(:item_type_mapping).and_return(ITEM_TYPE_MAPPING)
```

Comment:
```suggestion
```

Reply:
also `let(:hold) { build(:hold) }` on line 6 can be deleted as well, I think

Reply:
I think you are right, will change

Reply:
Oh I didn't fix this yet. Will do in next PR

Reply:
ahhh yes! I forgot that I was waiting on that :) in next year 👍